lib/twine/formatters/abstract.rb

```diff
@@ -27,7 +27,7 @@ def default_file_name
         raise NotImplementedError.new("You must implement default_file_name in your formatter class.")
       end
 
-      def set_translation_for_key(key, lang, value)
+      def set_translation_for_key(key, lang, value, section_name = @options[:section] || 'Uncategorized')
```

Comment:
Since this is core functionality, I'd prefer it to be tested. At least let's open an issue to keep track and add it later. Thoughts @scelis?